jira-slack-server-integration/jira-slack-server-integration-plugin/src/test/java/com/atlassian/jira/plugins/slack/util/CommentUtilTest.java

```diff
@@ -0,0 +1,51 @@
+package com.atlassian.jira.plugins.slack.util;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+
+class CommentUtilTest {
+    @ParameterizedTest
+    @MethodSource("validTagsProvider")
+    void testRemoveJiraTagsRemovesValidTags(String argument) {
+        assertEquals("some text", CommentUtil.removeJiraTags(argument));
+    }
+
+    static Stream<String> validTagsProvider() {
+        return Stream.of("{panel:title=My Title}some text{panel}",
```

Comment:
FWIW you could have used a Stream with two values (original and expected) so you would have just one dataset and there would be no need for hard coding the correct value.

An instance from [Stack Overflow](https://stackoverflow.com/questions/61483452/parameterized-test-with-two-arguments-in-junit-5-jupiter):

```

@ParameterizedTest
@MethodSource("provideParameters")
public void testParametersFromMethod(String name, int value) {
    System.out.println("method data " + name + " value " + value);
}

private static Stream<Arguments> provideParameters() {
    return Stream.of(
            Arguments.of("a", 1),
            Arguments.of("b", 2),
            Arguments.of("foo", 3)
    );
}
```

Reply:
Nice idea, i think i will use it next time